README.md

```diff
@@ -80,9 +80,8 @@ spec:
   replicas: 1
   server:
     distribution:
-      name: ollama
+      name: starter
     containerSpec:
-      port: 8321
       env:
       - name: INFERENCE_MODEL
```

Comment:
Shouldn't this be also `OLLAMA_INFERENCE_MODEL` like in the sample change below ?